tests/testsuite.src/run_misc.at

```diff
@@ -15682,3 +15682,159 @@ AT_CHECK([$COBCRUN_DIRECT ./prog6], [0],
 [01 02 03 04 05 06 07 08 09 ], [])
 
 AT_CLEANUP
+
+
+AT_SETUP([COB_SIGNAL_REGIME={0,1} registers runtime signal handler])
+AT_KEYWORDS([runmisc COB_SIGNAL_REGIME])
```

Comment:
cob_call_with_exception_check should also be included in the keywords

Reply:
Included.